src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/UnnecessarySemicolonAfterTypeMemberDeclarationCheck.java

```diff
@@ -29,6 +29,8 @@
 /**
  * <p>
  * Checks if unnecessary semicolon is used after type member declaration.
+ * Note: this check is not applicable to empty statements (unnecessary semicolons inside methods or
```

Comment:
@pbludov Should the note be in a note section?

Reply:
Forgot about it, changed